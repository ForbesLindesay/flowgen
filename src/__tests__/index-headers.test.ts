import {transformFile} from '../';

test('index.example.ts', () => {
  expect(transformFile(__dirname + '/index-headers.example.d.ts')).toMatchSnapshot();
})