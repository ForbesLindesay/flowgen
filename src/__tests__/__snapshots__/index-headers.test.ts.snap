// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.example.ts 1`] = `
"// @flow
// Generated using flowgen2

export type Foo = 'foo';
export type Bar = 'bar';
"
`;
